server/routes/induction/update/checkYourAnswersUpdateController.test.ts

```diff
@@ -0,0 +1,160 @@
+import type { SessionData } from 'express-session'
+import { NextFunction, Request, Response } from 'express'
+import createError from 'http-errors'
+import type { PageFlow, UpdateInductionQuestionSet } from 'viewModels'
+import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
+import { InductionService } from '../../../services'
+import CheckYourAnswersUpdateController from './checkYourAnswersUpdateController'
+import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
+import { aShortQuestionSetInductionDto } from '../../../testsupport/inductionDtoTestDataBuilder'
+import { aShortQuestionSetUpdateInductionDto } from '../../../testsupport/updateInductionDtoTestDataBuilder'
+
+jest.mock('../../../data/mappers/createOrUpdateInductionDtoMapper')
+
+describe('checkYourAnswersUpdateController', () => {
+  const mockedCreateOrUpdateInductionDtoMapper = toCreateOrUpdateInductionDto as jest.MockedFunction<
+    typeof toCreateOrUpdateInductionDto
+  >
+
+  const inductionService = {
+    updateInduction: jest.fn(),
+  }
```

Comment:
I can see that this is being done elsewhere too, but I don't think manually creating this mock is necessary?

You should be able to import a mocked version with `jest.mock()`. Eg:

`import InductionService from '../../../services/InductionService'`

`jest.mock('../../../services/InductionService')`

`const inductionService = new InductionService(null) as jest.Mocked<InductionService>`